Cliqz/Settings/CliqzAppSettingsTableViewController.swift

```diff
@@ -30,8 +30,9 @@ class CliqzAppSettingsTableViewController: AppSettingsTableViewController {
 		// Account Settings
 		#if PAID
 		let accountSettings = generateAccountSettings(prefs: prefs)
-        let accountSettingsTitle = NSLocalizedString("Lumen Account", tableName: "Cliqz" , comment: "[Settings] Account")
-        settings += [SettingSection(title: NSAttributedString(string: accountSettingsTitle), children: accountSettings)]
+		let accountSettingsTitle = NSLocalizedString("Lumen Account", tableName: "Cliqz" , comment: "[Settings] Account")
```

Comment:
you could just discard this commit

Reply:
I don't want to do all this rebase things with git :(